ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-bgp.j2

```diff
@@ -159,149 +159,159 @@
 
 | Neighbor | Remote AS | VRF | Shutdown | Send-community | Maximum-routes | Allowas-in | BFD | RIB Pre-Policy Retain |
 | -------- | --------- | --- | -------- | -------------- | -------------- | ---------- | --- | --------------------- |
-{%         for neighbor in router_bgp.neighbors | arista.avd.natural_sort %}
-{%             if router_bgp.neighbors[neighbor].peer_group is arista.avd.defined %}
+{%         for neighbor in router_bgp.neighbors | arista.avd.convert_dicts('ip_address') | arista.avd.natural_sort('ip_address') %}
+{%             if neighbor.peer_group is arista.avd.defined %}
 {%                 set inherited = namespace() %}
-{%                 if router_bgp.peer_groups[router_bgp.neighbors[neighbor].peer_group].remote_as is arista.avd.defined %}
-{%                     set inherited.remote_as = "Inherited from peer group " ~ router_bgp.neighbors[neighbor].peer_group %}
+{%                 set neighbor_peer_group = neighbor.peer_group %}
+{%                 set peer_groups = router_bgp.peer_groups | arista.avd.default([]) |
```

Comment:
Keep this variable singular as this holds only one entry of a reouter_bgp.peer_groups list.

Change ```peer_groups``` to ```peer_group```

Reply:
fixed